src/main/java/ch/njol/skript/Skript.java

```diff
@@ -1791,22 +1790,12 @@ static void logEx(final String... lines) {
 		for (final String line : lines)
 			SkriptLogger.LOGGER.severe(EXCEPTION_PREFIX + line);
 	}
-	
-	public static String SKRIPT_PREFIX = ChatColor.GRAY + "[" + ChatColor.GOLD + "Skript" + ChatColor.GRAY + "]" + ChatColor.RESET + " ";
-	
-//	static {
-//		Language.addListener(new LanguageChangeListener() {
-//			@Override
-//			public void onLanguageChange() {
-//				final String s = Language.get_("skript.prefix");
-//				if (s != null)
-//					SKRIPT_PREFIX = Utils.replaceEnglishChatStyles(s) + ChatColor.RESET + " ";
-//			}
-//		});
-//	}
-	
```

Comment:
Does it update the prefix when reloading the lang files without this code?

Reply:
The Message class from what I was told implemented this itself this code was commented out prior to this addition.
Switching this to a Message class will also allow in the future for a `/sk reload lang` to update language files ingame without a server restart